cocotb/share/lib/embed/gpi_embed.cpp

```diff
@@ -329,7 +320,7 @@ extern "C" int embed_sim_init(int argc, char const * const * argv)
     return ret;
 }
 
-extern "C" void embed_sim_event(gpi_event_t level, const char *msg)
+extern "C" COCOTB_EXPORT void _embed_sim_event(gpi_event_t level, const char *msg)
 {
     /* Indicate to the upper layer a sim event occurred */
```

Comment:
```suggestion
    /* Indicate to the upper layer that a sim event occurred */
```
While you are here...